### PR TITLE
Update Algolia section

### PR DESCRIPTION
**Pull request by jahrlin**
Signed-off-by: Joakim Ahrlin <joakim.ahrlin@gmail.com>

### CODE REVIEW

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve
/hold

A question and suggestion but this is an improvement as-is so I'd happily merge! Thanks!

File: README.md [hunk 1]
```diff
@@ -83,11 +83,13 @@ which is Algolia's free tool provided open-source projects.
 
 Configured through [a public JSON file](https://github.com/algolia/docsearch-configs/blob/master/configs/cert-manager.json).
```
[the remark this answers is not in the document]
Comment (jahrlin) [reply README.md:84]: True! I thought this was something else - let's remove it

File: README.md [hunk 2]
```diff
@@ -83,11 +83,13 @@ which is Algolia's free tool provided open-source projects.
...
-An account might be needed to trigger a re-indexing or possibly for other tasks. [`@munnerz`](https://github.com/munnerz/) has
-an account, which we might need to get access to.
+The crawler can be configured here: [https://crawler.algolia.com/admin/crawlers](https://crawler.algolia.com/admin/crawlers)
 
-The search API key is public, and is `e335fb22ca2fd2d96b4ba95b703430eb`. An example of use for the legacy docsearch v2 API is in
-[`search.js`](https://github.com/cert-manager/website/blob/6df6fd0986b5083a19fdbd597e2256f6627a274e/assets/js/search.js).
+The Algolia app can be configured here: [https://www.algolia.com/apps/01YP6XYAE7/dashboard](https://www.algolia.com/apps/01YP6XYAE7/dashboard)
+
+The Algolia API Key must be configured as an environment variable in Netlify.
+
```
[the remark this answers is not in the document]
Comment (jahrlin) [reply README.md]: Good idea, done!

**Review by SgtCoDFish — APPROVED** [review]
/lgtm
/approve
/unhold

Thanks, great improvement :+1:

**Pull request merged**